packages/components/addon/components/hds/pagination/nav/index.js

```diff
@@ -67,6 +67,14 @@ export default class HdsPaginationNavIndexComponent extends Component {
     return pages;
   }
 
+  get truncatedPages() {
+    console.log(
```

Comment:
left a console log.

(also, this function/getter is not need, see comment above)